app/views/katello/api/v2/repositories/base.json.rabl

```diff
@@ -57,3 +57,11 @@ end
 child :content_view => :content_view do |_repo|
   attribute :id, :name
 end
+
+child :content_view_version do
```

Comment:
For these added attributes you may want to take a look at the dev log when loading repositories API (both old and new) to make sure there are no n+1 queries. The 'bullet' gem logs that data for us. Would be good to pre-load these objects in the controllers in that case otherwise this could make things slow

Reply:
Is it logged by default? I'm not seeing anything about N+1 in my logs.